prow/cluster/jobs/istio/pkg/istio.pkg.master.yaml

```diff
@@ -0,0 +1,62 @@
+job_template: &job_template
```

Comment:
I still don't see this being used? Consider deleting.

Reply:
OK, deleted...